software/uptime-kuma/README.md

```diff
@@ -13,9 +13,16 @@ Egg is based on parkers generic nodejs egg
 Github: https://github.com/louislam/uptime-kuma
 
 ## Admin & login
-
 After installation and successful start, browse to your <ip>:<port> and setup the Admin.
 
+## Integrated Cloudflare Tunnel service
```

Comment:
this should not be needed in the readme. It is technicly some sort of web hosting

Reply:
> this should not be needed in the readme. It is technicly some sort of web hosting

Then Gitea is also Webhosting

Reply:
It is fine as it is running the integrated web server that comes with it. It is the part about contacting you for a difrend image. People come here to get working eggs and not everyone has discord. Also, only official images are supported as if they get your image, and it breaks we get to deal with the issue on the repo here

Reply:
fixed with last commit